xml/net_wicked.xml

```diff
@@ -530,7 +530,7 @@ ip route show</screen>
     (<literal>backup</literal>, <literal>install</literal>, etc.) in the
     resolver script. This in turn configures the DNS settings using
     <command>/sbin/netconfig</command>, or by manually writing
-    <filename>/etc/resolv.conf</filename> as a fallback.
+    <filename>/var/run/netconfig/resolv.conf</filename> as a fallback.
```

Comment:
This is the only occurrence of resolv.conf where I wholeheartedly agree to use /var/run because that's what netconfig actually uses.